config/routes.rb

```diff
@@ -9,4 +9,5 @@
   post "/login", to: "sessions#create"
   delete "/logout", to: "sessions#destroy"
   resources :users
+  resources :account_activations, only: [:edit]
```

Comment:
No need to add an array for only one action
`resources :account_activations, only: :edit`